# HG changeset patch
# User jwe
# Date 815117210 0
#      Tue Oct 31 05:26:50 1995 +0000
# Node ID dd087a402811c9d571d03737bdeae095fe3677bd
# Parent  54350c98055c5443b45bb13195cab6b832b16d53
[project @ 1995-10-31 05:26:50 by jwe]
Initial revision

diff --git a/scripts/specfun/betainc.m b/scripts/specfun/betainc.m
new file mode 100644
--- /dev/null
+++ b/scripts/specfun/betainc.m
@@ -0,0 +1,27 @@
+# Copyright (C) 1995 John W. Eaton
+# 
+# This file is part of Octave.
+# 
+# Octave is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 2, or (at your option) any
+# later version.
+# 
+# Octave is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with Octave; see the file COPYING.  If not, write to the Free
+# Software Foundation, 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+
+function y = betainc (x, a, b)
+  
+# Usage: betainc (x, a, b)
+#
+# See also: betai
+
+  y = betai (a, b, x);
+
+endfunction
diff --git a/scripts/specfun/gammainc.m b/scripts/specfun/gammainc.m
new file mode 100644
--- /dev/null
+++ b/scripts/specfun/gammainc.m
@@ -0,0 +1,27 @@
+# Copyright (C) 1995 John W. Eaton
+# 
+# This file is part of Octave.
+# 
+# Octave is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 2, or (at your option) any
+# later version.
+# 
+# Octave is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with Octave; see the file COPYING.  If not, write to the Free
+# Software Foundation, 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+
+function y = gammainc (x, a)
+  
+# Usage: gammainc (x, a)
+#
+# See also: gammai
+
+  y = gammai (a, x);
+
+endfunction
